crowbar_framework/app/models/node_object.rb

```diff
@@ -1249,6 +1249,22 @@ def poweroff
     bmc_cmd("power off")
   end
 
+  def service(service_name, action)
+    case service_name
+    when :chef
+      provider_service_object = ChefObject.get_provider_service_instance(self, "chef-client")
+    end
+
+    case action
+    when :start
+      provider_service_object.start_service
```

Comment:
@aspiers would be really nice if calling the correct instance method of `Chef::Provider::Service::*` just worked.
each provider service class has the right instance methods to start/stop/etc services.
but instead i get a permission error:
```
Expected process to exit with [0], but received '1'
---- Begin output of /sbin/service chef-client start ----
STDOUT:
STDERR: service: only root can use service
---- End output of /sbin/service chef-client start ----
Ran /sbin/service chef-client start returned 1
```
do you have an idea how to circumvent that?